examples/api/test/material/color_scheme/dynamic_content_color.0_test.dart

```diff
@@ -0,0 +1,178 @@
+    expect(find.text('onErrorContainer'), findsExactly(2));
+    expect(find.text('surface'), findsExactly(2));
+    expect(find.text('onSurface'), findsExactly(2));
+    expect(find.text('onSurfaceVariant'), findsExactly(2));
+    expect(find.text('outline'), findsExactly(2));
+    expect(find.text('shadow'), findsExactly(2));
+    expect(find.text('inverseSurface'), findsExactly(2));
+    expect(find.text('onInverseSurface'), findsExactly(2));
+    expect(find.text('inversePrimary'), findsExactly(2));
+
+    expect(loadColorSchemeCalls, hasLength(1));
+    expect(
+      loadColorSchemeCalls.single.$1,
+      isA<NetworkImage>()
+        .having(
+          (NetworkImage provider) => provider.url,
+          'url',
+          'https://flutter.github.io/assets-for-api-docs/assets/material/content_based_color_scheme_1.png',
+        ),
+    );
+    expect(
+      loadColorSchemeCalls.single.$2,
+      Brightness.light,
+    );
+
+    await tester.pumpAndSettle(); // Clears the timers from image loading.
+  });
+
+  testWidgets('The brightness can be changed', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      example.DynamicColorExample(
+        loadColorScheme: fakeColorSchemeLoader,
+      ),
+    );
+    await tester.pump();
+
+    expect(loadColorSchemeCalls, hasLength(1));
+    expect(
+      loadColorSchemeCalls.single.$1,
+      isA<NetworkImage>()
+        .having(
+          (NetworkImage provider) => provider.url,
+          'url',
+          'https://flutter.github.io/assets-for-api-docs/assets/material/content_based_color_scheme_1.png',
+        ),
+    );
+    expect(
+      loadColorSchemeCalls.single.$2,
+      Brightness.light,
+    );
+
+    await tester.tap(find.byType(Switch));
+    await tester.pump();
+
+    expect(loadColorSchemeCalls, hasLength(2));
+    expect(
+      loadColorSchemeCalls.last.$1,
+      isA<NetworkImage>()
+        .having(
+          (NetworkImage provider) => provider.url,
+          'url',
+          'https://flutter.github.io/assets-for-api-docs/assets/material/content_based_color_scheme_1.png',
+        ),
+    );
+    expect(
+      loadColorSchemeCalls.last.$2,
+      Brightness.dark,
+    );
+
+    await tester.pumpAndSettle(); // Clears the timers from image loading.
+  });
+
+  testWidgets('Tapping an image loads a new color scheme', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      example.DynamicColorExample(
+        loadColorScheme: fakeColorSchemeLoader,
+      ),
+    );
+    await tester.pump();
+
+    expect(loadColorSchemeCalls, hasLength(1));
+    expect(
+      loadColorSchemeCalls.single.$1,
+      isA<NetworkImage>()
+        .having(
+          (NetworkImage provider) => provider.url,
+          'url',
+          'https://flutter.github.io/assets-for-api-docs/assets/material/content_based_color_scheme_1.png',
+        ),
+    );
+    expect(
+      loadColorSchemeCalls.single.$2,
+      Brightness.light,
+    );
```

Comment:
Maybe make the test shorter by removing those lines as it was already tested on the other test and it is not what this test focus on.